src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -1635,6 +1660,14 @@ private void getCallInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_RESO
 
             pResult->methodFlags = FilterNamedIntrinsicMethodAttribs(pResult->methodFlags, methodToCall);
 
+            bool _TARGET_X86_ = _compilation.TypeSystemContext.Target.Architecture == TargetArchitecture.X86;
```

Comment:
```suggestion
```suggestion
```
            var targetDetails = _compilation.TypeSystemContext.Target;
            if (targetDetails.Architecture == TargetArchitecture.X86 
                && targetDetails.OperatingSystem == TargetOS.Windows
                && targetMethod.IsNativeCallable)
{
...
}
```

Reply:
`_TARGET_X86_` as variable name looks very non-CSharpy.

Reply:
That is done above, was following the existing style.

Reply:
Fortunately, being deleted in https://github.com/dotnet/runtime/pull/33274/files#diff-030b7d6ddb3dc6ea936a19a6fa55ac59L1475